{"name": "test_create_user_invalid_token", "status": "passed", "description": "Verify creating a user with an invalid token fails (401)", "attachments": [{"name": "Invalid Token Response", "source": "cd1d693d-d008-4b35-860b-286b741b3b6c-attachment.json", "type": "application/json"}], "start": 1757600673639, "stop": 1757600675547, "uuid": "a8708523-8ba5-4aa8-ae3c-42f993245bec", "historyId": "835ce4bbef170a855e338b21bdd139df", "testCaseId": "835ce4bbef170a855e338b21bdd139df", "fullName": "tests.test_users_negative#test_create_user_invalid_token", "labels": [{"name": "feature", "value": "User Management"}, {"name": "story", "value": "Negative Scenarios"}, {"name": "tag", "value": "negative"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_users_negative"}, {"name": "host", "value": "riju-Inspiron-15-3515"}, {"name": "thread", "value": "751743-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_users_negative"}], "titlePath": ["tests", "test_users_negative.py"]}